performance/src/main/java/uk/gov/gchq/palisade/example/perf/config/PerformanceConfiguration.java

```diff
@@ -125,13 +125,13 @@ public void setLiveRuns(final int liveRuns) {
 
     @Generated
     public List<String> getSkipTests() {
-        return skipTests;
+        return List.copyOf(skipTests);
```

Comment:
This getter is using the immutable and in a previous class the getter was using the unmodifiable.  Should it be changed?

Reply:
I've tried to make it so that when a Set/List is being set we take a copy of it but not sure if using the unmodifiable stuff would be better or if making all the setters and getters immutable would be better.

Reply:
I'm happy with the approach taken here